src/xSTIR/cSTIR/include/sirf/cSTIR/stir_data_containers.h

```diff
@@ -593,6 +593,24 @@ namespace sirf {
 			_data.reset(new Voxels3DF(*ad.get_proj_data_info_sptr()));
             this->set_up_geom_info();
 		}
+		STIRImageData
+		(const PETAcquisitionData& ad, const int nx, const int ny, const float zoom=1.F)
+		//! construct image with appropriate sizes for the given PETAcquisitionData
+		/*!
+			The number of z-planes is automatically determined in the STIR
+			implementation (see STIR documentation for VoxelsOnCartesianGrid),
+			but x and y size can be chosen. Voxel sizing is default_bin_size/zoom
+			(again, see STIR documentation for default_bin_size).
+		*/
+		{
+			const Coord3DF origin(0, 0, 0);
+			const Coord3DI sizes(-1, ny, nx);
```

Comment:
Does that -1 make sense @KrisThielemans ?

Reply:
yes, it selects the default number of slices. See https://github.com/UCL/STIR/blob/bfb4f82a4589dc8f77975ae44880094abd3d9640/src/include/stir/VoxelsOnCartesianGrid.h#L82-L87